src/fklearn/preprocessing/schema.py

```diff
@@ -117,6 +117,8 @@ def _learn(df: pd.DataFrame) -> LearnerReturnType:
                     columns_to_duplicate: Any = (kwargs[columns_to_bind] if columns_to_bind in
                                                  kwargs.keys() else
                                                  args[child_spec.args.index(columns_to_bind)])
+                    if isinstance(columns_to_duplicate, dict):
```

Comment:
What is the expected behaviour with this if statement?

Seems that the method `feature_duplicator` already solves the case if columns_to_duplicate is a dict(L53)

I've tested both with and without it and got the same results, do you think it is needed?

Reply:
Hmmmmmm it was failing here, but maybe I was testing on an older version, will check this out :thanks:

Reply:
Yeah it works, probably i'm just a dummy